lib/l10n/app_en.arb

```diff
@@ -129,6 +129,7 @@
   "usernamePatternError": "Value should only contain letters, digits, dashes and underscores.",
   "genericError": "Something went wrong",
   "wittyError": "It doesn't look like anything to me",
+  "toggleVisited": "Toggle Visited",
```

Comment:
I'd prefer this to be split into two strings, one for toggling on and one for toggling off, similar to how voting/favoriting/flagging works. I think "Mark read" and "Mark unread" describe the effect well and are analogous to, say, an email client. Also, please move it up a bit to group it with the other menu options: above `openParent` would work.